tests/test_patch_rebaser.py

```diff
@@ -510,7 +510,10 @@ def test_get_release_from_branch_name():
         "rhos-16.1-trunk-patches": "16.1",
         "rhos-90-trunk-patches": "90",
         "rhos-10.0-patches": "10.0",
-        "my_branch": "Unknown"
+        "my_branch": "Unknown",
+        "rhos-18.0-foo-trunk-patches": "18.0-foo",
+        "rhos-13.0-octavia-patches": "13.0-octavia",
+        "rhos-10.0": "Unknown"
```

Comment:
I have a doubt with this last case. Is this the expected  outcome?

Reply:
Hm... Fair point, it does look unsettling. Then again it's absolutely not a standard branch name for downstream patches and never has been in over a dozen releases, so it's probably fair.